src/gui/widgets/text_box_base.cpp

```diff
@@ -610,7 +640,7 @@ void text_box_base::signal_handler_sdl_key_down(const event::ui_event event,
 			break;
 
 		case SDLK_x:
-			if(!(modifier & modifier_key)) {
+			if( !(modifier & modifier_key) || !is_editable() ) {
```

Comment:
Though this sorta makes sense, we could also redirect Cut to Copy in a non-editable text field. Thoughts?

Reply:
Good idea.

Reply:
Implemented.